go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/prometheus/client_golang v1.2.1 // indirect
 	github.com/spf13/cobra v0.0.7
 	github.com/spf13/pflag v1.0.5
+        github.com/submariner-io/lighthouse v0.3.1-0.20200521062404-6d5bf9939fef
```

Comment:
```suggestion
       github.com/submariner-io/lighthouse v0.3.1-0.20200521062404-6d5bf9939fef
```

Reply:
Locally it shows fine for me but in PR it shows up with extra space.